tests/D2L.CodeStyle.Analyzers.Test/SpecRunner.cs

```diff
@@ -338,7 +338,10 @@ ImmutableArray<string> arguments
 
 					yield return new DiagnosticExpectation(
 						name: name,
-						arguments: arguments.Split( ',' ).Select( RemoveSingleLeadingSpace ).ToImmutableArray()
+						arguments: Regex.Split( arguments, @"(?<!\\\\)," )
+							.Select( arg => arg.Replace( "\\\\", string.Empty ) )

```

Comment:
I'm struggling to wrap my head around whether this should actually be `\\\\` or `\\`. Can you share an example of an expectation you've got working with this?

Reply:
I've tested with this and it works perfectly:
```cs
/* NonImmutableTypeHeldByImmutable(interface, SpecTests.Types.SomeGenericInterface<int\\, int>, ) */
```

Reply:
So given it's a comment, you don't actually need to be escaping the `\`. The following would be fine, and the regexes switched to `\\`. Sound reasonable?

`/* NonImmutableTypeHeldByImmutable(interface, SpecTests.Types.SomeGenericInterface<int\, int>, ) */`